hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/onset/OnsetResult.java

```diff
@@ -32,6 +32,7 @@
  */
 public record OnsetResult(
         @NonNull TransactionBody txBody,
+        @NonNull byte[] bodyBytes,
```

Comment:
In the javadoc  it will be clear to mention which workflow's onset result this is ?  If not please elaborate javadoc